ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/cco-manual-security-token-service/efs/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-efs-provision-chain.yaml

```diff
@@ -0,0 +1,26 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-efs-provision
+  steps:
+    - chain: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-provision
+    - chain: storage-conf-csi-optional-aws-efs
+  env:
+  - name: CONTROL_PLANE_INSTANCE_TYPE
+    default: "m6i.xlarge"
+    documentation: "Instance type for control plane nodes"
+  - name: COMPUTE_NODE_TYPE
+    default: "m5.xlarge"
+    documentation: "Instance type for compute nodes"
+  - name: ADD_INGRESS_RECORDS_MANUALLY
```

Comment:
I think we can remove these ENV vars in this chain, which are already set in cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-provision

Reply:
Sure, updated. Thanks!